public/javascript/scripts.js

```diff
@@ -0,0 +1,278 @@
+
+var globalInput = "";                   //input string will store the input typed in a keyboard up until the user presses enter
+var selectedInput = "none";             //currently selected input on the assentInfo table. e.x. 'asset_type' OR 'customer'
+var selectedTableAsset = "none";        //currently selected asset from the table. Will be set as the tag of the asset selected from the table. ex. 123456A
+
+var currentAsset = {};
+
+const MACRO_LETTERS = "ABYZ";            //Contains all of the recognized macro letter endings 
+const MACRO_LOAD_ASSET_LETTERS = "ABZ";  //Contains macro letters that will load an asset or create a new one if it is not in the db
+
+//variables that will hold HTML for different elements in the 
+var defaultAssetRow = "";
+var assetInfoNoChange = "";
+var assetInfoText = "";
+var assetInfoDate = "";
+var assetInfoNumber = "";
+var assetInfoSelect = "";
+
+$(document).ready(function() {
+    $(".signInBadge").focus();
+    
+    $(".signInBadge").keypress(function(e){
+        if(e.key == "Enter") {
+            if($(".signInBadge").val() == "123456Y") {
+                $(".backgroundCover").hide();
+                $(".signInContainer").hide();
```

Comment:
Right here it simply checks for the badge '123456Y' but eventually it will do an api call to the server to check whatever the user enters.